Tasks/MSBuild/MSBuild.ps1

```diff
@@ -97,7 +97,7 @@ if ($cleanBuild)
 $nugetPath = Get-ToolPath -Name 'NuGet.exe'
 if (-not $nugetPath -and $nugetRestore)
 {
-    Write-Warning "Unable to locate nuget.exe. Package restore will not be performed for the solutions"
+    Write-Warning (Get-LocalizedString -Key "Unable to locate nuget.exe. Package restore will not be performed for the solutions")
```

Comment:
to be consistent, you should pull out "nuget.exe" from the string and make it an arg.